Tribler/community/tunnel/tunnel_community.py

```diff
@@ -889,22 +883,27 @@ def on_cell(self, messages):
             if not self.relay_cell(circuit_id, message.payload.message_type, message):
 
                 plaintext, encrypted = TunnelConversion.split_encrypted_packet(message.packet, message.name)
+
+                # We don't know this circuit, so continue.
```

Comment:
There's also an else in the crypto_in method. That's used to decrypt relays, which are not circuits.